content/english/python-tensorflow/_index.md

```diff
@@ -41,12 +41,6 @@ When running your code for the first time, make sure to run your code by clickin
 
 {{% /notice %}}
 
-## Table of Contents
```

Comment:
We usually include a table of contents at the beginning of our workshops. Is there a particular reason why this was removed?

Reply:
There was nothing listed in the table of contents, I'll go ahead and leave it in though!

Reply:
wait, whoops it works perfectly fine! My mistake!